### ISSUE TITLE
Everyone updates their content on the website and searches for content they have lying around that could be hosted on the website.

### ISSUE THREAD

**Issue by nickhaf** [iqb-research/IQB-Methods#26]

**Comment by nickhaf** [comment]
- Verantwortliche: @franikowsp , @nickhaf 
- evtl. Einbindung in IQB-Webseite nötig
- Austausch mit Pauline Schröter dazu, wie das aussehen könnte (nächste Woche Termin)
- Eventuell Teilnahme an Schulungen zum Thema?

**Comment by nickhaf** [comment]
**Termin mit Pauline Schröter, @franikowsp und @nickhaf:**
- Erst einmal weitermachen wie bisher.
- Aktuell eher intern gedacht.
- Aber: für das IQB auch interessant das etwas größer aufzuziehen (eigene Workshops, Tutorials ...). Dafür müsste noch geklärt werden, was dann extern für welches Zielpublikum gestaltet werden soll, und was eher intern bleibt. 
- Auf der neuen Webseite wird es eine eigene Unterseite für das Methodenteam geben.
  - Dort könnte entweder zur GitHub-Seite verlinkt werden, oder irgendeine andere Art der Einbindung möglich gemacht werden.
  - Theoretisch müssten wir ja nicht auf GitHub hosten.
- @franikowsp und @nickhaf sprechen sich ab wer zu den zwei Tagen Workshop im April gehen will.

**Comment by nickhaf** [comment]
Vorschlag Pauline: IQB-Youtubekanal. Gäbe es von uns Bedarf/Material? -> bisher eher nicht

**Comment by mechtelm** [comment]
Ihr solltet den Inhalt dieses Repos nach [iqb-research.github.io](https://github.com/iqb-research/iqb-research.github.io) umziehen. Das war ja von mir damals nur ein Dummy, aber diese Quarto-Website ist ja umfassend für iqb-research gedacht und sollte nicht separat liegen.

Noch zu Repo-Benennung: Wir sind von Großbuchstaben weg, das ist umständlich, wenn man mal mit Handy unterwegs ist. Außerdem lassen wir stets "IQB" als Präfix weg, das könnte/müsste man ja sonst für alle Repos durchziehen - irgendwie überflüssig.

**Comment by nickhaf** [comment]
@nickhaf stellt Webseiten-Termin ein

**Comment by nickhaf** [comment]
## 15.03.24
### Material
- IQB-Vorgabe (z.B. Team ...)
- Shiny: 
  - Pakettutorials
  - Ergebnispräsentation
  - Lehre
  - https://docs.r-wasm.org/webr/latest/
  - https://quarto.org/docs/interactive/ojs/
-

**Comment by nickhaf** [comment]
Möglicherweise Hosting über HU-GitLab möglich mit privatem Bereich.

**Comment by mechtelm** [comment]
Mir ist noch nicht so klar, was Ihr mit dem privaten Bereich meint. Privat heißt eigentlich vertraulich, also Dinge, die andere außerhalb des IQB nicht wissen dürfen. Ansonsten würde ich Euch leidenschaftlich ermutigen, alles öffentlich zu machen, auch Halbfabrikate, Halbgares, auch Code und Fragen und Kommentare, die Ihr vielleicht später als peinlich einstufen würdet. Traut Euch! Man gewöhnt sich dran.

**Comment by nickhaf** [comment]
Ja: https://iqb-research.github.io/IQB-Methods/docs/FAQ/FAQ.html

**Comment by nickhaf** [comment]
> @nickhaf Exisitiert das Template eigentlich schon für die Methoden-FAQ?

Ja: https://iqb-research.github.io/IQB-Methods/docs/FAQ/FAQ.html

**Comment by nickhaf** [comment]
@sachseka stellt Termin mit @grewered und @nickhaf zur Besprechung der Struktur des Methoden-Reiters auf der Webseite ein.

**Comment by sachseka** [comment]
Ergebnisse Besprechung @sachseka mit @nickhaf und @grewered:

1. Texte für Toplevel und eatPakete
@sachseka macht Entwurf für Einführungstext:
	- Aufgabenbereiche (siehe Sprint-Planungstabelle) kurz beschreiben
	- Fokus nicht auf Paketen, aber als Aufgabenbereich mit nennen
	- Tabelle mit Personal, Link zu Personalseiten/Kontaktmöglichkeiten inklusive Zuständigkeiten -- oder vielmehr Zuordnung der Personen zu den Aufgabenbereichen, die die primäre Ordnung ausmachen (als Tabelle dann unten)

2. eatPakete-Seite
- @grewered zieht Sticker nach Pakethauptseite um, damit Teamhauptseite frei ist für Entwurf unter 1.
- Sticker und Tabelle beide toplevel, reinklicken geht sowohl über Sticker als auch über Tabelle nach Subseiten nach Github
- Namen mit Mitarbeiterseiten verlinken

3. Struktur des Rests
- @nickhaf  macht mal Entwurf für Ordnung aller bisherigen Unterpunkte

Nachgeordnet
4. DE/EN: idealerweise mit Umschaltbutton (erst nach Klärung mit PAS/Agentur)
- was EN ist von Edna dazu schwirrt zumeist noch was auf deutsch rum

5. Autorenfrage
- eher dafür, ggf unauffälliger
- nochmal besprechen, wenn wir Struktur besprechen

**Comment by nickhaf** [comment]
@franikowsp @grewered und @nickhaf setzen sich mal zur STruktur der Webseite zusammen. @nickhaf stellt Termin ein.

**Comment by sachseka** [comment]
Entwurf für Toplevel liegt hier: p:\Methoden\99_Arbeitsordner\webseite\Methoden2.docx
Bitte erstmal in der Word-Datei editieren/kommentieren @nickhaf @buchjani @grewered @apelalina @franikowsp @schumaki @beckerbenj denn da gibt es bestimmt noch einigen Absprachebedarf

**Comment by sachseka** [comment]
@sachseka fand Kommentare gut, wird so überarbeitet an Pauline übergeben

**Comment by sachseka** [comment]
Grundstruktur der vereinbarten Überarbeitung steht jetzt. Unter Learn müssen Strukturfragen noch ausdifferenziert werden; ggf. jede:r auch nochmal eigene Sachen anschauen. Help auch noch offen. Besprechen wir ggf. beim nächsten Mal mit Nicklas weiter.

Größere weitere Designfragen klären wir, wenn offizielles neues IQB-Design released ist.

**Comment by grewered** [comment]
04.11.24:
Learn page
- Kategorien rechts schaffen aktuell keine Übersicht
- Suchfunktion ergänzen 
- einzelne Pages erstellen: Anfänge, Advanced, Konventionen/Workflow, FAQ/statistische Fragen, R-SIG, etc.
